src/server/completions/completions.ts

```diff
@@ -46,16 +64,275 @@ export function getCompletionItems(
       },
     };
   });
+  return cleanedCompletions;
 }
 
-export function resolveCompletionItem(item: CompletionItem): CompletionItem {
-  item.detail = item.data.property;
-  const docs = (COMPLETION_DOCS[item.data.type] || {})[item.data.property];
-  if (docs) {
-    item.documentation = {
-      kind: MarkupKind.Markdown,
-      value: docs,
+async function getSchemaCompletions(
+  document: TextDocument,
+  context: CompletionParams,
+  translateCache: TranslateCache
+): Promise<string[]> {
+  const parse = parseDocumentText(document.getText(), context.position);
+  if (parse.nearestDefinitionText && parse.adjustedCursor) {
+    const exploreName = getQueriedExploreName(parse.nearestDefinitionText);
+    const {keyword, chain} = getQueryContext(
+      parse.nearestDefinitionText,
+      parse.adjustedCursor
+    );
+    if (exploreName !== null && keyword != null && chain !== null) {
+      let model: Model | undefined = undefined;
+      try {
+        model = await translateCache.translateWithTruncatedCache(
+          document,
+          parse.truncatedText,
+          parse.exploreCount,
+          exploreName
+        );
+      } catch (error: unknown) {
+        // eslint-disable-next-line no-console
+        console.error(
+          `Error fetching model for document sources and imports '${__filename}': ${error}`
```

Comment:
Check if you can use LogMessage instead.

Reply:
I checked with Will and it seems this isn't necessary as the console methods are bound the VSCode logging mechanisms